guides/development-workflow.md

```diff
@@ -448,6 +429,9 @@ have chosen the following existing style guides for our work:
   [Sass Guidelines - Syntax and formatting](https://sass-guidelin.es/#syntax--formatting)
 - JavaScript - [Standard](https://standardjs.com/)
 
+When joining an existing code base we suggest following any existing code style
+rather than switch over to ours.
+
 ---
 
 ## Pull requests
```

Comment:
I would love some attention on this section in particular.

Reply:
I think this is good. Other things I like to include:

- a recommendation for how to review (commit-by-commit or as-a-whole, ignoring whitespace changes, …)
- my thoughts on what's next, whether that's future refactorings, possible features, or when you expect this change to reach breaking point due to other changes in the future (eg "this is good for now, but after we've added a couple more WidgetServices we should plan to rewrite the Flooble system because …")

Reply:
By the contents of your feedback I'm going to guess it's to do with the code reviews rather than pull requests. I'll address them there and hope that's right 👍

Reply:
These are things I try to include as an author of a PR, rather than when code reviewing someone else's PR.